public/manifest.webextension-shim.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Should it be a JSONc file? (JSON with comment)

Reply:
for Android and iOS, need add new dependencies.